packages/react/src/Testimonial/Testimonial.module.css

```diff
@@ -1,12 +1,15 @@
 .Testimonial {
+  --testimonial-quote-size: var(--brand-text-size-300);
```

Comment:
Does this need to be a token? If so, could we prefix these tokens with `--brand` please for consistency with our other tokens. Would be great to also hoist it into our dedicated design tokens package please. Here's where they live: https://github.com/primer/brand/tree/main/packages/design-tokens/src/tokens/functional/components. You will need to create a new json source file there.